app/controllers/spree/base_controller_decorator.rb

```diff
@@ -1,17 +1,20 @@
-# TODO: should be just Spree::BaseController once base class is fixed here:
-# https://github.com/DynamoMTL/spree_chimpy/blob/master/app/controllers/spree/chimpy/subscribers_controller.rb#L1
-[ApplicationController, Spree::BaseController].each do |klass|
-  klass.class_eval do
-    after_filter :set_csrf_cookie_for_ng
+module Sprangular::BaseControllerDecorator
+  def self.prepended(base)
+    base.after_filter :set_csrf_cookie_for_ng
+  end
 
   protected
 
-    def set_csrf_cookie_for_ng
-      cookies['XSRF-TOKEN'] = form_authenticity_token if protect_against_forgery?
-    end
+  def set_csrf_cookie_for_ng
+    cookies['XSRF-TOKEN'] = form_authenticity_token if protect_against_forgery?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.